client/src/components/Perk.svelte

```diff
@@ -11,7 +11,7 @@
     <div class="text-container">
         <p>{text}</p>
     </div>
-    <button class="fancy-button">{buttonText}</button>
```

Comment:
If this is the Perk Card, we should display the name, as well as any conditional restrictions like: expiry date. Also the up/down votes should be displayed on the card.